src/makeRenderMethods.ts

```diff
@@ -0,0 +1,48 @@
+import ReactDOM from 'react-dom';
+
+// @TODO: Make this methods overridable via options
+export default function makeRenderMethods({
+  html,
+  pageElement,
+}: {
+  html: string;
+  pageElement: JSX.Element;
+}): {
+  renderHTML: () => { nextRoot: HTMLElement };
+  render: () => { nextRoot: HTMLElement };
+} {
+  // Update whole document content with SSR html
+  // @NOTE we have to preserve document.body element identity
+  // to not break @testing-library global "screen" object
+  function renderHTML() {
```

Comment:
I guess it's a bit confusing how this differs from `render`. Both render HTML at the end of the day Could we come up with a better name? For some reason Im thinking about `prerender`.

Reply:
The difference consists of `renderHTML` rendering just plain HTML in the DOM while `render` also binds React and eventListeners to it. Let's take a moment to find out better names 👍